test/integration/targets/cloudwatchlogs/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/aws
+unsupported
```

Comment:
Is there a reason these tests can't be supported? If additional policies need to be enabled those can be PR'ed to https://github.com/mattclay/aws-terminator. Looks like we'd also need a new terminator class for metric filters (the terminator is a lambda that auto-deletes resources after a period of time, in case a test or amazon fails in a way that leaves resources behind).

Reply:
Ok, should I open an issue there, so make sure it will no be forgotten?